routes/project/location/postgres.rb

```diff
@@ -308,6 +308,87 @@ class Clover
         response.headers["Content-Type"] = "application/x-pem-file"
         certs
       end
+
+      r.get "metrics" do
+        authorize("Postgres:view", pg.id)
+
+        start_ts = request.params["start"]&.to_i || Time.now.utc.to_i - 30 * 60
+        end_ts = request.params["end"]&.to_i || Time.now.utc.to_i
```

Comment:
Instead of integer, can we get data in some popular datetime format (i.e. something we can pass to Time.parse()) for example? That would be more user friendly I believe.

Reply:
I'll update this to support both timestamps and rfc3339 datetime strings